app/assets/stylesheets/common/admin/badges.scss

```diff
@@ -87,6 +87,30 @@
     .readonly-field {
       color: var(--primary-high);
     }
+
+    .badge-preview {
+      min-height: 110px;
+      max-width: 300px;
+      display: flex;
+      align-items: center;
+      background-color: var(--primary-very-low);
+      border: 1px solid var(--primary-low);
+      padding: 0 10px 0 10px;
+
+      img,
+      svg {
+        width: 60px;
+        height: 60px;
+      }
+
+      .badge-display-name {
+        margin-left: 5px;
+      }
+    }
+
+    .badge-required {
+      font-weight: bold;
+    }
```

Comment:
Lift and shift from being nested in `.award-badge`, which is also used for button styling, into `.current-badge`, which is the shared body class for this UI element.